tmt/cli.py

```diff
@@ -1288,35 +1333,31 @@ def clean(context: click.core.Context, **kwargs: Any) -> None:
 @dry
 def clean_runs(
         context: click.core.Context,
-        path: str,
+        workdir_root: str,
         last: bool,
         id_: str,
         keep: int,
         **kwargs: Any) -> None:
     """
     Clean workdirs of past runs.
 
-    Remove all runs in /var/tmp/tmt by default. Path to where runs
-    should be searched can be specified using PATH argument.
+    Remove all runs in '/var/tmp/tmt' by default.
     """
-    echo(style('clean', fg='red'))
     defined = [last is True, id_ is not None, keep is not None]
     if defined.count(True) > 1:
         raise tmt.utils.GeneralError(
             "Options --last, --id and --keep cannot be used together.")
     if keep is not None and keep < 0:
         raise tmt.utils.GeneralError("--keep must not be a negative number.")
-    if not os.path.exists(path):
-        raise tmt.utils.GeneralError(f"Path '{path}' doesn't exist.")
-    exit_code = 0
-    if not tmt.Clean(parent=context.obj.clean, context=context).runs():
-        exit_code = 1
-    raise SystemExit(exit_code)
+    if not os.path.exists(workdir_root):
+        raise tmt.utils.GeneralError(f"Path '{workdir_root}' doesn't exist.")
+    clean_obj = tmt.Clean(parent=context.obj.clean, context=context)
+    context.obj.clean_partials["runs"].append(functools.partial(clean_obj.runs))
```

Comment:
Why are you applying `functools.partial()` here but not feeding it with any partial arguments?

Reply:
After chat with @FrNecas agreed to remove it in a6f2bcb as it does not seem to be needed.